robot-tests/keywords.robot

```diff
@@ -71,11 +71,15 @@ Set environment to
     Set Global Variable    ${REGULAR_USER_PASSWORD}          ${REGULAR_USER_PASSWORD_${env}}
     Set Global Variable    ${API_USER_USERNAME}              ${API_USER_USERNAME_${env}}
     Set Global Variable    ${API_USER_PASSWORD}              ${API_USER_PASSWORD_${env}}
-    Set Global Variable    ${BUILDLY_REACT_TEMPLATE_BASE_URL}            ${BUILDLY_REACT_TEMPLATE_BASE_URL_${env}}
+<<<<<<< HEAD
+    Set Global Variable    ${BUILDLY_REACT_TEMPLATE_BASE_URL}              ${BUILDLY_REACT_TEMPLATE_BASE_URL_${env}}
+=======
+    Set Global Variable    ${BUILDLY_REACT_TEMPLATE_BASE_URL}             ${BUILDLY_REACT_TEMPLATE_BASE_URL_${env}}
+>>>>>>> replace-buildly-ui
```

Comment:
Please, remove one of the global variables, they are duplicated

Reply:
I changed it. Let me know there are any mistakes